classes/class-wc-connect-service-settings-store.php

```diff
@@ -199,7 +199,9 @@ public function get_enabled_services() {
 		 * @return object|array
 		 */
 		public function get_service_settings( $service_id, $service_instance = false ) {
-			return get_option( $this->get_service_settings_key( $service_id, $service_instance ), array() );
+			$settings = get_option( $this->get_service_settings_key( $service_id, $service_instance ), array() );
+
+			return $settings;
```

Comment:
No reason to make this change (I guess to put some debugging code here and forgot to change it back :smile: )